x-pack/plugins/alerting/server/plugin.ts

```diff
@@ -604,10 +605,14 @@ export class AlertingPlugin {
       encryptedSavedObjectsClient,
       eventLogger: this.eventLogger!,
       executionContext: core.executionContext,
-      getMaintenanceWindowClientWithRequest,
       getRulesClientWithRequest,
       kibanaBaseUrl: this.kibanaBaseUrl,
       logger,
+      maintenanceWindowsService: new MaintenanceWindowsService({
+        cacheInterval: this.config.rulesSettings.cacheInterval,
```

Comment:
this is used just for reducing the cache interval for testing so I reused the setting from the rules settings service.